ui-meta/beegfs.yml

```diff
@@ -0,0 +1,25 @@
+name: "beegfs"
+label: "BeeGFS"
+description: "BeeGFS storage cluster."
+
+parameters:
+  - name: "beegfs_mds_volume_size"
+    label: "Metadata volume size"
+    description: "BeeGFS metadata server volume size."
+    kind: "integer"
+    options:
+      min: 2
+
+  - name: "beegfs_oss_volume_size"
+    label: "Storage volume size per node"
```

Comment:
Maybe add a unit, i.e. "... (GB)"

Reply:
The per-node bit can probably live in the description, i.e. "Storage volume size" and "Size of the BeeGFS object storage volume on each node."